metadata-db-app/int-test/cmr/metadata_db/int_test/concepts/association_save_test.clj

```diff
@@ -0,0 +1,21 @@
+(ns cmr.metadata-db.int-test.concepts.association-save-test
+  "Contains integration tests for saving tool associations. Tests saves with various
```

Comment:
should this be "tool" associations or is it "generic" associations

Reply:
generic - I have fixed the description.